httpx/config.py

```diff
@@ -61,6 +62,10 @@ def __init__(
             verify = True
             self._load_client_certs(ssl_context)
 
+        if trust_env:
+            if verify is True or verify is None:
+                verify = get_ssl_cert_file()  # type: ignore
```

Comment:
This doesn't look quite right to me, but perhaps I've got myself in a muddle.

If verify is True, but `SSL_CERT_FILE` is not set, then `get_ssl_cert_file` will return `None`, which looks like a behavioral change from what we've got right now.

Reply:
I also think we should *probably* inspect the environment at the point of calling `load_ssl_context`, rather than on `__init__`.